### PR TITLE
T03: Fibonacci Series

### PR DESCRIPTION
**Pull request by Dakshita-Minocha**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,7 +1,16 @@
 ﻿namespace Training {
    internal class Program {
       static void Main(string[] args) {
-         Console.WriteLine("Hello, World!");
+         Console.Write("Enter no of terms to be displayed");
```
Comment (v-priya) [inline Program.cs:4]: End with a colon and space so display and user input is clear.

File: Program.cs [hunk 2]
```diff
@@ -1,7 +1,16 @@
 ﻿namespace Training {
    internal class Program {
       static void Main(string[] args) {
-         Console.WriteLine("Hello, World!");
+         Console.Write("Enter no of terms to be displayed");
+         int.TryParse (Console.ReadLine(), out int input);
+         int item1 = 0, item2 = 1;
```
Comment (v-priya) [inline Program.cs:6]: Use better variable names. The name should convey the purpose and meaning of the variable.

File: Program.cs [hunk 3]
```diff
@@ -1,7 +1,16 @@
 ﻿namespace Training {
    internal class Program {
       static void Main(string[] args) {
-         Console.WriteLine("Hello, World!");
+         Console.Write("Enter no of terms to be displayed");
+         int.TryParse (Console.ReadLine(), out int input);
+         int item1 = 0, item2 = 1;
+         Console.Write(item1 + " " + item2 + " ");
```
Comment (v-priya) [inline Program.cs:7]: Not formatted

**Review by v-priya — CHANGES_REQUESTED** [review]

File: Program.cs [hunk 4]
```diff
@@ -1,7 +1,16 @@
 ﻿namespace Training {
    internal class Program {
-      static void Main(string[] args) {
-         Console.WriteLine("Hello, World!");
+      static void Main (string[] args) {
+         Console.Write ("Enter no of terms to be displayed: ");
+         int.TryParse (Console.ReadLine (), out int input);
+         int num1 = 0, num2 = 1;
```
Comment (v-priya) [inline Program.cs:6]: `num1` and `num2` are not bad. But, wouldn't something like `prev` and `current` make more sense?

**Review by v-priya — CHANGES_REQUESTED** [review]

File: Program.cs (hunk 4 above)
Comment (Dakshita-Minocha) [reply Program.cs:6]: wouldn't the sum of num1 &2 be the current number though?

Comment (v-priya) [reply Program.cs:6]: Yes, you're right. Since you are using the variable `sum`, this makes sense than prev and current.

**Pull request closed without merging**